#include <ros/ros.h>

#ifndef __ICREATE_TRANSPORTATION_SINGLE_TRANSPORTATION
#define __ICREATE_TRANSPORTATION_SINGLE_TRANSPORTATION

namespace icreate {

class SingleTransportation {
    public:
        SingleTransportation();

        ~SingleTransportation();

        void 


    private:

    public:

    private:


};

}